integration/test/test.sh

```diff
@@ -4,5 +4,13 @@ echo "Dir : $SCRIPT_DIR"
 cd $SCRIPT_DIR/test
 tag=${1?Specify valid test tag}
 export PATH=$PATH:/usr/local/go/bin
+if [ ! -n "$VERSION" ]; then
+  echo "VERSION not supplied."
+  VERSION="$(date +%Y.%m.%d%H | cut -c 3-)"
```

Comment:
date and hour are mixed
```
hardikl@hardikl-mac-0 harvest % date +%Y.%m.%d%H | cut -c 3-
22.02.0313
```

Reply:
That's mostly for internal versioning. Same as what we have in nightly builds.